src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -9,6 +9,10 @@ Main.createListener=Invalid output format. Found ''{0}'' but expected ''{1}'' or
 #
 Main.errorCounter=Checkstyle ends with {0} errors.
 Main.loadProperties=Unable to load properties from file ''{0}''.
+MetadataGeneratorLogger.emptyOutputStream=Output stream should be empty
+MetadataGeneratorLogger.unexpectedClose=Unexpected close count
+MetadataGeneratorLogger.unexpectedFlush=Unexpected flush count
+MetadataGeneratorLogger.violationRequiresException=Violation should contain exception message
```

Comment:
@romani @nrmancuso This is a test only class and the new messages are now being put into production files to resolve this? Is this something we want to do?

This test class was not listed in the issue that I found.

Reply:
>Is this something we want to do?

Absolutely not.

>This test class was not listed in the issue that I found.

@MANISH-K-07 why are we doing this update for a test class?

Reply:
@rnveach , @nrmancuso , 
Sorry, I have misunderstood the Issue requirement.
I will reuse the PR to update other hardcoded exceptions and messages of Checker.java

Again, really sorry for this

Reply:
Internal or test classes can continue to be English only